blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineStepImpl.java

```diff
@@ -272,18 +272,7 @@ private Object convert(String name, ParameterValue v) throws IOException, Interr
         }
     }
     private boolean canSubmit(InputStep inputStep){
-
-        Authentication a = Jenkins.getAuthentication();
-        String submitter = inputStep.getSubmitter();
-        if (submitter==null || a.getName().equals(submitter)) {
-            return true;
-        }
-        for (GrantedAuthority ga : a.getAuthorities()) {
-            if (ga.getAuthority().equals(submitter)) {
-                return true;
-            }
-        }
-        return false;
+        return inputStep.canSubmit();
```

Comment:
InputStep.canSubmit() is marked `@Deprecated`, these methods have been moved over to InputStepExecution, but they are privates so can't use. I thought its better to copy that code instead of using deprecated ones in  InputStep as its going to lag behind vs how it evolves in InputStepExecution. Right fix is to get them made public in InputStepExecution and use it directly. 

I am fine using deprecated ones in the meantime.